app/controllers/cart_items_controller.rb

```diff
@@ -30,4 +30,12 @@ def decrement
 
         redirect_to controller: 'carts', action: 'index'
     end
+
+    def destroy
+        product_id = params[:id]
+        cart_item = current_user.cart.cart_items.find_by(product_id: product_id)
+        cart_item.destroy!
```

Comment:
What if cart_item failed to destroy.
Have some alert.

Reply:
Ya will add a rescue block for errors and a flash message to show